### DISCUSSION TITLE
Non-mammalian model use

### DISCUSSION THREAD

**Discussion by jayjsmith** [kanaverse/kana#76] [Ideas]
Hello, 

I'm wondering if kana can or will be able to be used for/with non-mammalian model scRNA-seq data, like _C. elegans_. Such a useful tool - I'd like to try it out with my nematode data but I'm not sure whether ref genome can be changed or mitochondrial gene list for qual control can be updated to use _elegans_ gene names. 

Thanks!

**Comment by LTLA** [comment]
Shouldn't be too hard. Might even be possible right now if your symbols start with `mt-` or something similar, in which case you can disable the default gene list and use the prefix search instead. Otherwise we'll just have to add the corresponding Ensembl IDs + symbols to the mitochondrial list. That should be the only change; the rest of the app doesn't care what the reference genome is.

**Reply by jayjsmith** [reply]
Great! Unfortunately _C. elegans_ mitochondrial gene names don't start with "mt-", BUT I do have the gene names + Ensembl ID used for previous publication(s):

_nduo-1_ WBGene00010959
_nduo-2_ WBGene00010961
_nduo-3_ WBGene00010966
_nduo-4_ WBGene00010963
_nduo-5_ WBGene00010967
_nduo-6_ WBGene00010957
_ctc-1_ WBGene00010964
_ctc-2_ WBGene00010965
_ctc-3_ WBGene00010962
_ndfl-4_ WBGene00010962
_atp-6_ WBGene00010960
_ctb-1_ WBGene00000829

**Reply by LTLA** [reply]
Interesting, I didn't realize Ensembl was using Wormbase for its C. elegans identifiers. Nice.

FWIW I get the same list from Ensembl 104. Plus some tRNAs and rRNAs; I'll throw these into the list as well.

```
> ens <- ahub[["AH95683"]] # i.e., Ensembl 104 for C. elegans.
> info <- select(ens, keys=keys(ens), keytype="GENEID", columns=c("GENEID", "SYMBOL", "SEQNAME", "GENEBIOTYPE"))
> is.mito <- info$SEQNAME %in% c("MT", "M", "chrM", "chrMT", "MtDNA")
> info[is.mito,]
              GENEID  SYMBOL SEQNAME    GENEBIOTYPE
673   WBGene00000829   ctb-1   MtDNA protein_coding
8990  WBGene00010957  nduo-6   MtDNA protein_coding
8991  WBGene00010958  ndfl-4   MtDNA protein_coding
8992  WBGene00010959  nduo-1   MtDNA protein_coding
8993  WBGene00010960   atp-6   MtDNA protein_coding
8994  WBGene00010961  nduo-2   MtDNA protein_coding
8995  WBGene00010962   ctc-3   MtDNA protein_coding
8996  WBGene00010963  nduo-4   MtDNA protein_coding
8997  WBGene00010964   ctc-1   MtDNA protein_coding
8998  WBGene00010965   ctc-2   MtDNA protein_coding
8999  WBGene00010966  nduo-3   MtDNA protein_coding
9000  WBGene00010967  nduo-5   MtDNA protein_coding
12254 WBGene00014450  MTCE.1   MtDNA           tRNA
12255 WBGene00014451  MTCE.2   MtDNA           tRNA
12256 WBGene00014452  MTCE.5   MtDNA           tRNA
12257 WBGene00014453  MTCE.6   MtDNA           tRNA
12258 WBGene00014454  MTCE.7   MtDNA           rRNA
12259 WBGene00014455  MTCE.8   MtDNA           tRNA
12260 WBGene00014456  MTCE.9   MtDNA           tRNA
12261 WBGene00014457 MTCE.10   MtDNA           tRNA
12262 WBGene00014458 MTCE.13   MtDNA           tRNA
12263 WBGene00014459 MTCE.14   MtDNA           tRNA
12264 WBGene00014460 MTCE.15   MtDNA           tRNA
12265 WBGene00014461 MTCE.17   MtDNA           tRNA
12266 WBGene00014462 MTCE.18   MtDNA           tRNA
12267 WBGene00014463 MTCE.19   MtDNA           tRNA
12268 WBGene00014464 MTCE.20   MtDNA           tRNA
12269 WBGene00014465 MTCE.22   MtDNA           tRNA
12270 WBGene00014466 MTCE.24   MtDNA           tRNA
12271 WBGene00014467 MTCE.27   MtDNA           tRNA
12272 WBGene00014468 MTCE.28   MtDNA           tRNA
12273 WBGene00014469 MTCE.29   MtDNA           tRNA
12274 WBGene00014470 MTCE.30   MtDNA           tRNA
12275 WBGene00014471 MTCE.32   MtDNA           tRNA
12276 WBGene00014472 MTCE.33   MtDNA           rRNA
12277 WBGene00014473 MTCE.36   MtDNA           tRNA
```
